### PR TITLE
feat: SmartPassIllustration component

### PR DESCRIPTION
**Pull request by mainframev**
<br/><br/><br/><url>LiveURL: https://orbit-feat-smart-pass-illustration.surge.sh</url>

### CODE REVIEW

**Comment by mainframev** [comment]
- We shoud pick some names for images, now i just named them as `v1, v2, v3 ...` :)

File: packages/orbit-components/src/SmartPassIllustration/index.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    );
+
+  if (name === "v3")
+    return (
+      <SmartPassV3 name={name} primary={primary} height={height} secondary={secondary} {...props} />
+    );
+
+  return (
+    <SmartPassV4 name={name} primary={primary} height={height} secondary={secondary} {...props} />
+  );
```
Comment (silvenon) [inline packages/orbit-components/src/SmartPassIllustration/index.js:47]: Instead I recommend something like this:

```jsx
import * as Illustrations from "./components"

// ...

const Illustration = Illustrations[`SmartPass${name.toUpperCase()}`]
if (Illustration) {
  return (
    <Illustration name={name} primary={primary} height={height} secondary={secondary} {...props} />
  );
}
return null;
```

File: packages/orbit-components/src/index.js.flow [hunk 2]
```diff
@@ -205,6 +206,7 @@ declare export var ServiceLogo: ServiceLogoType;
 declare export var Stepper: StepperType;
 declare export var StepperStateless: StepperStatelessType;
 declare export var StopoverArrow: StopoverArrowType;
+declare export var SmartPassIllustration: SmartPassIllustrationType;
```
Comment (silvenon) [inline packages/orbit-components/src/index.js.flow:209]: Should be added to `.d.ts` file as well, no?

File: packages/orbit-components/src/SmartPassIllustration/__tests__/index.test.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        alt={alt}
+        description={description}
+        ariaLabelledby={ariaLabelledby}
+        name="v1"
+      />,
+    );
+    expect(screen.getByTestId(dataTest)).toBeInTheDocument();
+    expect(screen.getByText(description)).toBeInTheDocument();
+    expect(screen.getByRole("img")).toHaveAttribute("alt", alt);
+    expect(screen.getByRole("img")).toHaveAttribute("aria-labelledby", "aria-labeled");
```
Comment (silvenon) [inline packages/orbit-components/src/SmartPassIllustration/__tests__/index.test.js:28]: Maybe also check `<title>`? I wonder if `ByTitle` will work for this case.

**Review by silvenon — CHANGES_REQUESTED** [review]
Also delete `.DS_Store` and add it to `.gitignore`.

File: packages/orbit-components/src/SmartPassIllustration/components/SmartPassV1.js [hunk 4]
```diff
@@ -0,0 +1,490 @@
...
+  dataTest,
+}: SvgProps) => {
+  return (
+    <svg
+      height={height}
+      viewBox="0 0 2200 1300"
+      fill="none"
+      data-test={dataTest}
+      role="img"
+      alt={alt || name}
```
[the remark this answers is not in the document]
Comment (mainframev) [reply packages/orbit-components/src/SmartPassIllustration/components/SmartPassV1.js:29]: yes, forgot to remove it

File: packages/orbit-components/src/SmartPassIllustration/components/SmartPassV2.js [hunk 5]
```diff
@@ -0,0 +1,272 @@
...
+  dataTest,
+}: SvgProps) => {
+  return (
+    <svg
+      height={height}
+      data-test={dataTest}
+      viewBox="0 0 2200 1300"
+      fill="none"
+      role="img"
+      aria-labelledby={ariaLabelledby}
```
Comment (silvenon) [inline packages/orbit-components/src/SmartPassIllustration/components/SmartPassV2.js:29]: I don't know what passing `ariaLabelledby` to `SmartPassIllustration` and forwarding it to SVG elements does, but I assume you researched it and know that it's the right thing to do. 🤷‍♂️

**Review by silvenon — CHANGES_REQUESTED** [review]

File: packages/orbit-components/.gitignore [hunk 6]
```diff
@@ -3,11 +3,14 @@ umd
...
 .out
 .cache
+.DS_STORE
+src/**/.DS_STORE
 src/data/airportIllustrations.json
 src/data/illustrations.json
 src/data/icons.json
 src/Illustration/index.js.flow
 src/AirportIllustration/index.js.flow
+src/SmartPassIllustration/.DS
```
[the remark this answers is not in the document]
Comment (mainframev) [reply packages/orbit-components/.gitignore:13]: i dont know, i did not put it there :D

**Review by silvenon — APPROVED** [review]

**Pull request merged**